packages/metro-config/src/ExpoMetroConfig.ts

```diff
@@ -69,7 +71,7 @@ export function getDefaultConfig(
       },
     },
     transformer: {
-      babelTransformerPath: require.resolve('../transformer'),
+      babelTransformerPath: require.resolve('metro-react-native-babel-transformer'),
```

Comment:
Couldn't get the custom transformer to work for now, for some reason when using it something goes wrong and I start getting "missing class properties transform" errors... I think we'll want to do it as a follow up, though (it allows removing `babel.config.js` while defaulting to `babel-preset-expo`).

Reply:
definitely would be worth adding.